### DISCUSSION TITLE
Weird noise

### DISCUSSION THREAD

**Discussion by eehealy** [simonsobs/sodetlib#390] [Q&A]
After running uxm_setup, we have some weird features in the noise PSD plots. There is a large bump around 9 Hz and some higher f features as well. Have folks seen this sort of noise before? Any suggestions for debugging the cause? The AMC 1 in the plot is one that we haven't used before, but we have used AMC 0 without issue and it seems to have similar features to AMC 1 in Band 0

![image](https://github.com/simonsobs/sodetlib/assets/33061247/b7cfde6a-04f6-434d-9f8c-3c6931093330)

**Comment by msilvafe** [comment]
The peaks look like some modulated 60Hz noise. Some follow up Q's to help debug. What does this look like in the timestream? Is it a continuous wave sort of pickup or a digital looking pickup? Is it aligned along bias lines in addition to Smurf bands? Do you have any FR or BL shorts?

**Comment by jlashner** [comment]
Additionally, that bump at around 8 Hz looks like it could be HWP pickup. If you can recreate this PSD with a higher nperseg, you may be able to resolve that better.

**Reply by msilvafe** [reply]
I think no HWP in the LATRt system so likely not the culprit...

**Reply by jlashner** [reply]
ah right I was confusing this with SAT2

**Reply by jlashner** [reply]
I do still think that a higher nperseg would be useful for understanding that peak ~10 Hz

**Comment by eehealy** [comment]
Ok, so we confirmed that there is a short on UFM-Lp2's BL-1, which explains the 60 Hz pickup on Band 0. The FRs and all other BLs are ok, i.e. no shorts to ground.

The plot above was from this weekend, here's a plot from running relock this morning. As you can see, it's a bit different that what we saw on Friday.

<img width="1121" alt="image" src="https://github.com/simonsobs/sodetlib/assets/33061247/0c186b78-bcbf-41d8-b6b2-62a2fe64957d">

But mystery solved! This was a new AMC we were using, so it had never been plugged into the timing system.. When I made that connection the noise got a lot better:
<img width="1119" alt="image" src="https://github.com/simonsobs/sodetlib/assets/33061247/a94b134b-1da1-498b-b580-39b065ddadf4">

**Comment by eehealy** [comment]
This might warrant a new thread, but next I need to figure out why we have so many detectors normal when we think our focal plane is at 100mK. Going to dig into previous testing of these UFMs.

Thanks again to everyone for the help :)

**Reply by msilvafe** [reply]
No problem, glad you figured this out! Yes, start a new thread with some more info on how things are setup optically, thermally, which UFMs and how the testing went at Pton etc. and we can try to help there.
